src/requirements/reqs-functions.ts

```diff
@@ -38,11 +32,11 @@ function createRequirementJSON(
     required: requirement.minCount,
     description: requirement.description,
     source: requirement.source,
-    fulfilled: null,
+    fulfilled: undefined,
```

Comment:
What's the reason we are setting fulfilled as undefined rather than null?

Reply:
So we have one less case to worry about. Usually the convention is to either completely ban `undefined` or completely ban `null`